JS/posts.js

```diff
@@ -0,0 +1,71 @@
+const API_URL_base = "https://api.noroff.dev/api/v1";
+const postUrl =  API_URL_base + "/social/posts?_author=true";
+
+import { createPosts } from "./createposts.js";
+
+
+const postContainer = document.getElementById("postContainer");
+
+const queryParams = new URLSearchParams(window.location.search);
+const filterByLikes = queryParams.get("_reactions") === "true";
+const filterByComments = queryParams.get("_comments") === "true";
+
+
+async function getPosts() {
+    try {
+        const getPostsData = {
+            method: "GET",
+            headers: {
+                Authorization: `Bearer ${localStorage.getItem("accessToken")}`,
+            },
+        };
+
+        const response = await fetch(postUrl,getPostsData);
+        const json = await response.json();
+        console.log(json);
+        postContainer.innerHTML = "";
+
+        // Filter and sort the posts
+        let filteredPosts = json;
+
+        // Sort the filtered posts based on reactions (likes) if filterByLikes is true
+        if (filterByLikes) {
+            filteredPosts = filteredPosts.filter(post => post._count.reactions > 0);
```

Comment:
@Noroff-Anders I can see you are filtering by reactions and comments. Good job! However, can you filter by tag using the `_tag` parameter as mentioned in the API doc at https://docs.noroff.dev/social-endpoints/posts#filtering? 

What do you think @Fermain

Reply:
The tag filter is more useful for creating a theme for the social site, for example only showing posts with "cars" tag to make an automotive themed social site. This is fine :)